product_abc_classification_sale_stock/models/abc_classification_profile.py

```diff
@@ -59,15 +59,15 @@ def _get_all_product_ids(self):
         self.env.cr.execute(
             """
             SELECT
-                abc_rel.product_id
+                product_id
             FROM
-                abc_classification_profile_product_rel abc_rel
+                abc_classification_profile_product_rel
```

Comment:
This should not be changed.... These changes fixes an issue. see https://github.com/OCA/product-attribute/pull/1723